appcues/build.gradle

```diff
@@ -83,7 +83,7 @@ dependencies {
     implementation "androidx.browser:browser:1.4.0"
     implementation "androidx.startup:startup-runtime:1.1.1"
     implementation "androidx.navigation:navigation-compose:2.5.3"
-    implementation "androidx.compose.ui:ui:$compose_version"
+    api "androidx.compose.ui:ui:$compose_version"
```

Comment:
extending AbstractComposeView require us to change here from implementation to api, meaning we are including the compose dependency in our graph, this is not a big issue since people should already have compose implemented. just noting here the reason why this was changed.

Reply:
does this open any concern about version conflicts with the version we are using versus the customer app? Wondering if we can conceal this by just having an `AppcuesFrameView` that is a specific `View` that we provide that internally contains the compose view?